include/wx/webview.h

```diff
@@ -220,9 +234,17 @@ class WXDLLIMPEXP_WEBVIEW wxWebView : public wxControl
     //Find function
     virtual long Find(const wxString& text, int flags = wxWEBVIEW_FIND_DEFAULT) = 0;
 
+    //Establish EmulationLevel for RunScript IE
```

Comment:
I don't think these methods are necessary in the base class and I'd make them specific to IE backend.

In fact, I'm not sure why do we want to have `GetEmulationLevel()` at all, especially as it creates questions about what should be returned for unknown emulation level values etc. Do you really see any realistic scenarios in which this method would be useful? I just don't.

As for `SetEmulationLevel()`, it seems to be potentially useful, although I think that it doesn't need to be so precise, i.e. I'd basically keep just the possibilities to use the default (compatibility, old) mode and the new mode in which JS output works. And I'd probably use the latter by default, unless there is some really good reason not to.